### DISCUSSION TITLE
WFCM Multivendor feature

### DISCUSSION THREAD

**Discussion by mathiou005** [inspireui/support#5673] [New Feature/Ideas Requests]
Suggestion:

Hello, pls when you won’t integrite Abandoned Cart Recovery plugins to mobile app? 

Like suggestions : you should add this plugin too, wich is very compatible with WCFM Marketplace; 

Plugin name===> https://codecanyon.net/item/woocommerce-group-buy-and-deals/18977834

Thanks

**Comment by jooooo-dot** [comment]
Up!!
